src/Compatibility/Microsoft.DotNet.ApiCompatibility/ApiDiffer.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.CodeAnalysis;
+using Microsoft.DotNet.ApiCompatibility.Abstractions;
+using System.Collections.Generic;
+
+namespace Microsoft.DotNet.ApiCompatibility
+{
+    public class ApiDiffer
```

Comment:
The use of the term `Diff` and it's variants `Differ` `Diffing` feels  a bit confusing and feel like jargon.  Other options would be `Difference` or `Compare`/`Comparer`/`Comparison`.

Reply:
I guess I like `ApiComparer` best here... `ApiDifference` might clash with `CompatDifference` in the sense that if you read them both could mean the same?